consensus/consensus.md

```diff
@@ -374,7 +374,7 @@ TemperedTotalTime = 0.75 * N * DesiredTimeBetweenBlocks + 0.2523 * TotalTime
 ```
 
 Where FTL = Future Time Limit - i.e. the time a block is allowed to be
-"from the future". We use 10 minutes (600 s).
+"from the future". We use 9 minutes (540 s).
```

Comment:
A little note like "We use 9 minutes (540 s) i.e. 3 times the DesiredTimeBetweenBlocks." would help the reader though optional.

Reply:
It's updated now.